packages/react-devtools-shared/src/backend/profilingHooks.js

```diff
@@ -53,6 +69,12 @@ if (supportsUserTimingV3) {
   performanceTarget = performance;
 }
 
+// Some environments (e.g. React Native / Hermes) don't support the performance API yet.
+const getCurrentTime =
+  typeof performance === 'object' && typeof performance.now === 'function'
+    ? () => performance.now()
+    : () => Date.now();
```

Comment:
Curious how accurate this is in getting the current time (also thinking about this for the transition tracing stuff) and if it matters?

Reply:
Not sure if this is what you're asking but we stub out the timer for DevTools tests so that it reads from `unstable_now` (in `scheduler/src/forks/SchedulerMock`) so that tests are repeatable.